pkgs/development/python-modules/tidalapi/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, requests
+, ffmpeg-python
+, keyring
+, pillow
+, pytest
```

Comment:
Use `pytestCheckHook` instead.

```suggestion
, pytestCheckHook
```

Reply:
I tried to use it, but the tests failed because they try to connect to the internet, which is forbidden in the sandbox.

Reply:
Disable the individual tests using `disabledTests`.

Reply:
> Disable the individual tests using `disabledTests`.

I analyzed test by test and all require internet access

Reply:
Then add a comment to `doCheck` that says so.